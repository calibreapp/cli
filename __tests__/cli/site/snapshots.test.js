const { runCLI } = require('../../utils')

test('missing site argument', () => {
  return runCLI({ args: 'site snapshots', testForError: true }).then(stdout => {
    expect(stdout).toMatchSnapshot()
  })
})

test('missing site argument value', () => {
  return runCLI({ args: 'site snapshots --site', testForError: true }).then(
    stdout => {
      expect(stdout).toMatchSnapshot()
    }
  )
})
