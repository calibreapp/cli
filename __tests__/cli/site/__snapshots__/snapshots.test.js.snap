// Jest Snapshot v1, https://jestjs.io/docs/snapshot-testing

exports[`site snapshots lists all snapshots 1`] = `
"2 snapshots
ID | URL                                                | REF | CLIENT | STATUS                       
14 | http://localhost:3000/calibre/calibre/snapshots/14 |     |        | Completed 12:00AM 6-Feb-2019 
13 | http://localhost:3000/calibre/calibre/snapshots/13 |     |        | Completed 12:22AM 30-Jan-2019
To see snapshots after 13, run: calibre site snapshots --site=calibre --cursor=MjU=="
`;
